modules/nf-core/seqfu/derep/meta.yml

```diff
@@ -0,0 +1,44 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "seqfu_derep"
+description: Dereplicate FASTX sequences printing the total members per unique sequence, can dereplicate dereplicated FASTA files summing the numbers of members
```

Comment:
What exactly does dereplicate dereplicated FASTA files mean? I don't understand.

Reply:
See https://telatin.github.io/seqfu2/tools/derep.html and https://rcedgar.github.io/usearch12_documentation/cmd_fastx_uniques.html

This strange term is quite standard in amplicon analysis, will try to clarify in the description :)

Reply:
Ok, that is fine, clearer now.